Jellyfin.Server.Implementations/Trickplay/TrickplayManager.cs

```diff
@@ -80,7 +81,12 @@ public TrickplayManager(
     public async Task MoveGeneratedTrickplayDataAsync(Video video, LibraryOptions libraryOptions, CancellationToken cancellationToken)
     {
         var options = _config.Configuration.TrickplayOptions;
-        if (!CanGenerateTrickplay(video, options.Interval, libraryOptions))
+        if (!CanGenerateTrickplay(video, options.Interval))
+        {
+            return;
+        }
+
+        if (libraryOptions is null || !libraryOptions.EnableTrickplayImageExtraction)
```

Comment:
shouldn't we check for that before anything else to avoid any unnessesary operation?